crates/sqlite-libsql/src/lib.rs

```diff
@@ -1,16 +1,18 @@
-use libsql_client::DatabaseClient;
 use spin_world::sqlite::{self, RowResult};
 
 #[derive(Clone)]
 pub struct LibsqlClient {
-    client: libsql_client::reqwest::Client,
+    inner: libsql_client::http::Client,
 }
 
 impl LibsqlClient {
-    pub fn new(url: String, token: String) -> Self {
-        Self {
-            client: libsql_client::reqwest::Client::new(url, token),
-        }
+    pub fn create(url: &str, token: String) -> anyhow::Result<Self> {
```

Comment:
(for my own education, not a problem)  Why does this change from `new` to `create` - is it because returning a `Result` is unidiomatic for `new`?

Reply:
Yes exactly - I believe clippy has a (maybe off by default) warning for this.